RelayTestApp/Assets/Scripts/States/ConnectingGameState.cs

```diff
@@ -53,6 +53,8 @@ IEnumerator WaitForResponse()
             }
             yield return new WaitForFixedUpdate();
         }
+
+        yield return new WaitForSeconds(2);
```

Comment:
I added the 2 seconds because whenever I leave match or lobby and then look for a new lobby it would get stuck on the loading screen. I suspect it needed abit more time for the server to deal with the request before I look again.

Reply:
Ah I understand, I think that's fine but wondering what happens if let's say the server takes longer than 2 seconds to respond to the leave action, very unlikely but still. Is this just to wait on a response from the server? Or is it waiting on something happening server side that we get no confirmation of?